python3/libexec/probe-device-for-file

```diff
@@ -41,15 +43,15 @@ def stderr_redirected(to=os.devnull):
 
 if __name__ == "__main__":
     if len(sys.argv) != 3:
-       print("Usage: %s <device> <file>" % sys.argv[0])
-       sys.exit(2)
-    device = sys.argv[1]
-    file = sys.argv[2]
+        print("Usage: %s <device> <file>" % sys.argv[0])
+        sys.exit(2)
+        device = sys.argv[1]
+        file = sys.argv[2]
     try:
         # CA-316241 - fsimage prints to stderr
         with stderr_redirected(to="/dev/null"):
-            fs = xenfsimage.open(device, 0)
-            if fs.file_exists(file):
+            fs = xenfsimage.open(device, 0)#type:ignore
+            if fs.file_exists(file):#type:ignore
```

Comment:
What's the error being ignored? Are you sure it's not relevant?

Reply:
Error which is being ignored is "probe-device-for-file:53:34 - error: "device" is unbound (reportUnboundVariable) and probe-device-for-file:54:31 - error: "file" is unbound (reportUnboundVariable) " which occurs when the variable named device and file  that has not been defined. 

Code snippet,
if __name__ == "__main__":
    if len(sys.argv) != 3:
        print("Usage: %s <device> <file>" % sys.argv[0])
        sys.exit(2)
        **device = sys.argv[1]**
        **file = sys.argv[2]**

I can initialize device and file to None to fix this issue.

Reply:
Yes, please initialize the variables, even if it's None

Reply:
Previously :
>pyright probe-device-for-file

/home/ashwinh/xapichange/xen-api/python3/libexec/probe-device-for-file
  /home/ashwinh/xapichange/xen-api/python3/libexec/probe-device-for-file:55:34 - error: "device" is unbound (reportUnboundVariable)
  /home/ashwinh/xapichange/xen-api/python3/libexec/probe-device-for-file:56:31 - error: "file" is unbound (reportUnboundVariable)
2 errors, 0 warnings, 0 informations

Now:
>pyright probe-device-for-file
0 errors, 0 warnings, 0 informations